compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -1373,13 +1373,14 @@ pub enum VariantDiscr {
 }
 
 #[derive(Debug, HashStable, TyEncodable, TyDecodable)]
-pub struct FieldDef {
+pub struct FieldDef<'tcx> {
     pub did: DefId,
     pub name: Symbol,
     pub vis: Visibility<DefId>,
+    pub value: Option<ty::Const<'tcx>>,
```

Comment:
At that stage of the representation, should we stay with a simple `Option<DefId>>` to represent the anonymous constant?